proposals/34:upgrade-10/upgradeHelpers.js

```diff
@@ -16,29 +16,35 @@ export const printKeys = async () => {
   console.log('========== GOVERNANCE KEYS ==========');
```

Comment:
JS shouldn't ever have to read these files from disk. they're in the ENV and thus in JS constants already.

however, this section wasn't necessary to port to JS. I think it would be best rn to cut lines 16-49.

Reply:
Got it, thanks for the background, just deleted this section